src/w3d/renderer/texturebase.h

```diff
@@ -100,6 +100,7 @@ class TextureBaseClass : public RefCountClass
     bool Is_Initialized() const { return m_initialized; }
     void Set_Dirty(bool dirty) { m_dirty = dirty; }
     unsigned Get_Width() { return m_width; }
+    MipCountType Get_Mip_Level_Count() { return m_mipLevelCount; }
```

Comment:
These Get functions can be `const` safely.

Reply:
fixed.